postgresql/database_test.go

```diff
@@ -57,32 +52,37 @@ var settings = ConnectionURL{
 	Password: password,
 }
 
-var host = flag.String("host", "testserver.local", "Testing server address.")
+var host = flag.String("host", "localhost", "Testing server address.")
```

Comment:
I think it would be better to keep `testserver.local` and manage actual resolution in `/etc/hosts`, the problem with `localhost` is that it always resolves to `127.0.0.1` while a name like `testserver.local` can be pointed either to `127.0.0.1`, a machine on a LAN or to a VM in the same physical host.